apps/landingPage/components/Demo.js

```diff
@@ -5,12 +5,7 @@ import { Heading, Button } from '@kiwicom/orbit-components/lib/';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 import styled from 'styled-components';
 
-import {
-  github,
-  tequila,
-  documentation,
-  codeKiwi,
-} from '../../../../linksConfig';
+import { github, tequila, documentation, codeKiwi } from '../../../linksConfig';
```

Comment:
Should this config be moved to `@kiwicom/margarita-config`?

Reply:
I guess it should, will move now